### PR TITLE
Self validating structured data and removal of transaction managers

### PR DESCRIPTION
**Pull request by dirvine**
Please feel free to comment on initial proposal.  An issue will be created to discuss this more widely in terms of design and improvements.

### CODE REVIEW

**Comment by dirvine** [comment]
Be good if @chandraprakash  or @qi.ma could comment and help with this one. Be good to get it in shape and then have some discussions and debates if it makes merge and then has an issue assigned for all to chime in.

**Comment by dirvine** [comment]
Paging @s-brian  as Shepherd (make sure PR is in order to get pushed and proposed). Step after this is to assign an issue number and begin any discussions

File: proposed/0000-Remove-Transaction-Managers.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+# Detailed design
+
+The design entails reducing all StructuredData types to two sub-types, therefore it should be able to
+be recognised by the network as StructuredData and all such sub-types handled exactly in the same manner. the sub types are defined here:
+
+##FixedStructuredData
+
+```
+struct FixedStructuredData {
+type : TagType, // 64 Bytes
```
Comment (ustulation) [inline proposed/0000-Remove-Transaction-Managers.md]: 64 bits i think (= 8 bytes)

**Comment by mmoadeli** [comment]
Below are two minor issue

1- 'Feature Name: Remove Transaction Managers from network and have network only recognise 2 Structured data sub-types'
The first part of the feature name 'Remove Transaction Managers from network' is only an  outcome of the second part 'network only recognise 2 Structured data sub-types'. and also the first part of the feature name is not mentioned anywhere in text. I would put the first part of the feature name, 'Remove Transaction Managers from network' as an Expected outcome, not in title.

2- In 'New Protocols' , the first sentence is not parse-able by my English level.

**Comment by dirvine** [comment]
Thanks @mmoadeli  I will make those changes

**Comment by dirvine** [comment]
Changes applied

**Pull request merged**